guides/doc-Upgrading_and_Updating/topics/upgrading_overview.adoc

```diff
@@ -32,9 +32,9 @@ Upgrading {Project} takes approximately 1{range}2 hours.
 Upgrading {SmartProxy} takes approximately 10{range}30 minutes.
 
 * Ensure that you have sufficient storage space on your server.
-For more information, see {InstallingProjectDocURL}Preparing_your_Environment_for_Installation_{project-context}[Preparing your Environment for Installation] in _Installing {ProjectServer} from a Connected Network_ and {InstallingSmartProxyDocURL}preparing-environment-for-capsule-installation[Preparing your Environment for Installation] in _Installing {SmartProxyServer}_.
+For more information, see {InstallingProjectDocURL}Preparing_your_Environment_for_Installation_{project-context}[Preparing your Environment for Installation] in _{InstallingServerDocTitle}_ and {InstallingSmartProxyDocURL}preparing-environment-for-capsule-installation[Preparing your Environment for Installation] in _Installing {SmartProxyServer}_.
```

Comment:
This can potentially be handled using the attribute in case a build target want to differentiate between connected and disconnected setups.

Reply:
I'm not sure I get your point, but the Installing Proxy DocTitle should be replaced on this line as well.